includes/class-micropub-endpoint.php

```diff
@@ -90,22 +90,35 @@ public static function register_route() {
 	public static function load_auth() {
 		static::$micropub_auth_response = apply_filters( 'indieauth_response', static::$micropub_auth_response );
 		static::$scopes                 = apply_filters( 'indieauth_scopes', static::$scopes );
+		// Every user should have this capability which reflects the ability to access your user profile and the admin dashboard
+		if ( ! current_user_can( 'read' ) ) {
+			return new WP_Error( 'forbidden', 'Unauthorized', array( 'status' => 403 ) );
+		}
+
+		// If there is no auth response this is cookie authentication which should be rejected
```

Comment:
minor nit: mind explaining why?

Reply:
You shouldn't be able to log into Micropub without a token

Reply:
I probably should use the LOCAL AUTH variable

Reply:
> You shouldn't be able to log into Micropub without a token

right, sorry, i was hoping you could add that to the comment. maybe ideally with a link to the relevant part of the spec. thanks!

Reply:
Was planning on it. But wanted to make sure you got the meaning